src/node.cc

```diff
@@ -4473,6 +4492,11 @@ inline int Start(Isolate* isolate, IsolateData* isolate_data,
   if (debug_enabled)
     EnableDebug(&env);
 
+  if (load_napi_modules) {
+    ProcessEmitWarning(&env, "N-API is an experimental feature "
+        "and could change at any time.");
+  }
+
```

Comment:
@jasongin I’d say take it out, and we can re-visit that in the unlikely case anybody complains.